design/v6-bundle/project/styles_v6_agents_grid.css

```
@@ -0,0 +1,247 @@
+@import url('styles_v6.css');
```

Stylelint (17.7.0)

[error] Expected "url('styles_v6.css')" to be "'styles_v6.css'" (import-notation)

(import-notation)